### PR TITLE
Add `get graph` which plots the DAG.

### PR DESCRIPTION
**Pull request by bethune-bryant**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [x] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  

## Description

It may be useful to have a plot command.

### CODE REVIEW

**Comment by bnemanich** [comment]
I think this looks pretty cool! You will need to add `matplotlib` to the dependencies list in `setup.py` and the conda recipe.

File: open_ce/get_graph.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from open_ce.inputs import Argument
+from open_ce import graph
+from open_ce import build_env
+from open_ce import utils
+
+COMMAND = "graph"
+
+DESCRIPTION = 'Plot a dependency graph.'
+
+ARGUMENTS = build_env.ARGUMENTS + \
```
Comment (bnemanich) [inline open_ce/get_graph.py:31]: Should this be `inputs.ENV_BUILD_ARGS` instead of `build_env.ARGUMENTS`?

File: open_ce/get_graph.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            [Argument.WIDTH,
+             Argument.HEIGHT]
+
+def export_graph(args):
+    '''Entry Function'''
+    build_tree = build_env.construct_build_tree(args)
```
Comment (bnemanich) [inline open_ce/get_graph.py:37]: Are there other places in our code that `construct_build_tree` can be called?

Comment (bethune-bryant) [reply open_ce/get_graph.py:37]: Probably, that was part of the reason for me making it a separate function. I had started looking into that, but then got side tracked. I'll look into using it other places too.

File: open_ce/get_graph.py (hunk 1 above)
Comment (bethune-bryant) [reply open_ce/get_graph.py:31]: I had that at first, but I want to make sure that if we update the build_env args they get updated here too. I want it to use the exact same args as build_env.

Comment (bnemanich) [reply open_ce/get_graph.py:31]: OK, that is fine. There will just be some arguments that don't make as much sense such as `--skip_build_packages`, but that should be fine.

Comment (bethune-bryant) [reply open_ce/get_graph.py:31]: After going back and looking at it, you're right, I had it backwards.

File: open_ce/get_graph.py (hunk 2 above)
Comment (bethune-bryant) [reply open_ce/get_graph.py]: Okay, I moved it and I'm using it in all the places we were making a BuildTree.

**Review by bnemanich — APPROVED** [review]

**Pull request merged**